### PR TITLE
Allow different chunk stores

### PR DESCRIPTION
**Pull request by AaronVasquez**

### CODE REVIEW

File: hooknode/services/s3.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+type S3 struct{}
+
+// ChunkStore interface
+func (i *S3) AdaptReqChunks(chunks []types.ReqChunk) []types.IotaChunk {
+	result := make([]types.IotaChunk, len(chunks))
+	// TODO: Fetch Message from s3
+	for i, reqChk := range chunks {
+		result[i] = types.IotaChunk{
+			Address: reqChk.Address,
```
Comment (rfornea) [inline hooknode/services/s3.go:13]: If we're storing the hash in s3, would the address be null as well, prior to fetching?

Comment (AaronVasquez) [reply hooknode/services/s3.go:13]: Yep, going to make all those changes in a future PR. Wanted this one to be just a refactoring.

Comment (rfornea) [reply hooknode/services/s3.go:13]: Okay sounds good

**Review by rfornea — APPROVED** [review]

**Comment by AaronVasquez** [comment]
Weird that this one has 4 travis builds triggered...

**Pull request merged**